frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -34,7 +34,7 @@ export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
           title="A bucket will be created to provide the OCS Service."
           isInline
         />
-        <p className="co-legend co-required ceph-ocs-desc__legend">
+        <p className="co-legend co-required ceph-ocs-co-required">
           Select at least 3 nodes in different failure domains with minimum requirements of 16 CPUs
```

Comment:
Ideally `co-required` should be on `Select Nodes` since thats the form field, the rest is info text. https://github.com/openshift/console/blob/ae4892fab3dea439738463fc982e4d1575f32024/frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx#L23-L25

Reply:
Its as per the UXD.

Reply:
Since most of the required form field dont have this css, lets remove it.
Thanks!